### PR TITLE
Part 5: Update external-text and app components concerning notes, references, climatologies and variables

### PR DESCRIPTION
**Pull request by QSparks**
This PR includes updates to various text-based aspects of the application. These changes are part of the larger fnlf-cmip6-release.

The changes to [src/components/app-root/App/App.js](https://github.com/pacificclimate/plan2adapt-v2/pull/268/files#diff-262dd5d8a02ccc32ad3c299e4e960a3b51050bae109e8994942f2a3cbc5c119a) and [src/assets/time-periods.js](https://github.com/pacificclimate/plan2adapt-v2/pull/268/files#diff-fc2110d20428425a46ae8d7beeeb8b56c8e4f823c53531fc692d40e710cea813) are so that the new baseline dataset and the new 2030s future climatology are used. All remaining changes apply to [public/external-text/default.yaml](https://github.com/pacificclimate/plan2adapt-v2/pull/268/files#diff-3a64bad0e5151b24a64fd0b3884e472f170f4358c8162de576879f1c24cc3f4b)

Key changes include:
- Replace `1970s` baseline climos with `1990s`
- Replace `2020s` climos with `2030s`
- Replace `tasmean` variable with `tas`
- Replace `prsn` variable with `snow`
- Update `References` and fix all broken external page links
- Updated resources for cmip6
- Added current staff to team page   
- Update summary tab prologue
- Add a column for the historical `baseline mean` value and units
- Remove units from precip and snow variables in the climate variable column
- Update summary tab notes and add threshold definitions for degree-day climate indices
- Disable the now redundant notes tab
- Add additional seasons to the statistics table for precip and snow variables
- Update `pr` and `snow` data units to `mm/day` and tune the range of values for the colormap scale of the climate variable, ncWMS layers on the maps tab 
- Add units and precision spec for baseline means of climate variables

Demo link: https://services.pacificclimate.org/dev/plan2adapt/app/

### CODE REVIEW

**Comment by corviday** [comment]
Is there a demo we can play with?

**Comment by QSparks** [comment]
> Is there a demo we can play with?

There is! I'll add it to the PR description.
Demo link: http://beehive.pacificclimate.org/plan2adapt/app

**Comment by QSparks** [comment]
Marked as a draft while I add a few new changes requested by RCI: [978d7f2...0eb87c7](https://github.com/pacificclimate/plan2adapt-v2/compare/978d7f2...0eb87c7)

File: public/external-text/default.yaml [hunk 1]
```diff
@@ -400,55 +412,57 @@ tabs:
             annual:
               displayUnits: mm/year (cum)
               range:
-                min: 10
-                max: 4000
+                min: 200
```
Comment (jameshiebert) [inline public/external-text/default.yaml:415]: Can you update the ticks to make sure that they are a superset of the included range (e.g. add 6000 below). It feels a bit odd to have ticks defined for values outside of the defined range (e.g. < 200), but OTOH it makes it easier to revert to a lower range later on.

File: public/external-text/default.yaml [hunk 2]
```diff
@@ -500,13 +514,12 @@ tabs:
       This figure shows the range of projected change in
       ${season} ${variable.label} for ${region.properties.english_na}
       over three time periods
-      (2020s, 2050s, and 2080s), according to a PCIC-standard set of
-      GCM projections (see 'Notes' tab for more information).
+      (2020s, 2050s, and 2080s), according to the PCIC12 set of downscaled GCM projections from CanDCS-M6 under the SSP5-8.5 emissions scenario.
```
Comment (jameshiebert) [inline public/external-text/default.yaml:517]: Should "2020s" say "2030s"?

File: public/external-text/default.yaml [hunk 3]
```diff
@@ -744,30 +769,39 @@ tabs:
 
       - label: Team
         cards:
-          - header: '[Trevor Murdock](https://pacificclimate.org/about-pcic/people/trevor-murdock)'
```
Comment (jameshiebert) [inline public/external-text/default.yaml:747]: Trevor should still be credited here on P2A. You can remove the link to his bio, and credit him with being the "Founding concept designer".

File: src/components/app-root/App/App.js [hunk 4]
```diff
@@ -27,7 +27,7 @@ import SummaryTabBody from '../SummaryTabBody';
 import ImpactsTabBody from '../ImpactsTabBody';
 import MapsTabBody from '../MapsTabBody';
 import GraphsTabBody from '../GraphsTabBody';
-import NotesTabBody from '../NotesTabBody';
+// import NotesTabBody from '../NotesTabBody';
```
Comment (jameshiebert) [inline src/components/app-root/App/App.js:30]: Just remove instead of commenting it out?

File: src/components/app-root/App/App.js [hunk 5]
```diff
@@ -232,10 +232,10 @@ export default class App extends Component {
           variableOpt: this.state.variableOpt,
         },
       },
-      notes: {
-        Component: NotesTabBody,
-        props: {},
-      },
+      // notes: {
```
Comment (jameshiebert) [inline src/components/app-root/App/App.js:235]: Likewise.

**Review by jameshiebert — APPROVED** [review]
If RCI is happy with the text changes, then they work for me. I went through line-by-line and confirmed that the deployed demo works as written here. Just a few really minor comments from me, but overall it looks great. Many thanks!

**Pull request merged**